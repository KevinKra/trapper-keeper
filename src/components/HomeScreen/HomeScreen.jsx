import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';
import ListArea from '../../containers/ListArea/ListArea';
// import ListInput from '../../containers/ListInput/ListInput';

const HomeScreen = () => {
	return (
		<Fragment>
			<Link to="/new-note">
				<button>Add New List</button>
			</Link>
			<ListArea />
		</Fragment>
	);
};

export default HomeScreen;
