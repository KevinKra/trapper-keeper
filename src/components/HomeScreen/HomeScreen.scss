@import '../../variables.scss';

.HomeScreen {
  min-height: 100vh;
  max-width: 1300px;
  margin: auto;
}

.btn-area {
	height: 300px;
	width: 100%;
	display: flex;
	justify-content: center;
	align-items: center;

	& button {
		height: 50px;
		width: 200px;
		font-size: 20px;
		font-weight: 500;
		border: 0;
		color: white;
		background-color: $btnGreen;
    border-radius: 3px;
    
    &:hover {
      cursor: pointer;
      background-color: lighten($color: $btnGreen, $amount: 2%)
    }
	}
}
