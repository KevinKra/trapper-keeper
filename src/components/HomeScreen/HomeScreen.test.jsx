import React from 'react';
import HomeScreen from './HomeScreen';
import { shallow } from 'enzyme';

describe('HomeScreen', () => {
	let wrapper;
	beforeEach(() => {
		wrapper = shallow(<HomeScreen />);
	});

	it('should match the snapshot', () => {
		expect(wrapper).toMatchSnapshot();
	});
});

// TODO test that button routes to /new-note?