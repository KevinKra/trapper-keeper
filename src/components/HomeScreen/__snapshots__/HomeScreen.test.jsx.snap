// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen should match the snapshot 1`] = `ShallowWrapper {}`;
